afh-api/Controllers/MovieController.cs

```diff
@@ -39,7 +37,7 @@ public async Task<ActionResult<Movie>> GetMovieById(int id)
             return movie;
         }
 
-        [HttpPost("AddMovie")]
+        [HttpPost("{id}")]
```

Comment:
I don't think this one should have an id passed in, as the ID is only created once the movie's added to the database. Otherwise looks good!

Reply:
In the UserController, you used `Add` so I copied the convention but could also just be blank, leaving the path as `localhost:5089/movie/`

Reply:
@halimahexe this has been updated!